components/raftstore/src/store/worker/metrics.rs

```diff
@@ -160,6 +160,11 @@ lazy_static! {
         &["cf"]
     )
     .unwrap();
+    pub static ref FULL_COMPACT: Histogram = register_histogram!(
```

Comment:
Recommend to display this newly introduced metric in `metrics/grafana/tikv_details.json`, and place it into `Storage` or `Cluster` section, like following shows:
![image](https://github.com/tikv/tikv/assets/18441614/466f4291-30f9-497f-93c7-d6096bae0e2f)

Reply:
Cool, I was manually setting this up, was looking where to place it. Will do.

Reply:
Actually, since this is an experimental feature for now, I'll do this in a follow up PR. I think it's very important to add ability to pause/resume for now.